packages/clay-css/src/scss/functions/_global-functions.scss

```diff
@@ -77,8 +77,7 @@
 @function math-sign($num) {
 	@if (type-of($num) == 'number') {
 		@return -($num);
-	}
-	@elseif (type-of($num) == 'string') {
+	} @else if (type-of($num) == 'string') {
```

Comment:
@pat270 the only question regarding what I'm concerned about is if it might fail when the files are built using `node-sass` instead of using `sass` (dart-sass)?

I'm thinking here more in case we ever have to backport a version of Clay with this modification to an older version of DXP that is still using `node-sass`.

Reply:
Oh perfect, thanks @pat270!